Lib/fontTools/varLib/varStore.py

```diff
@@ -5,7 +5,6 @@
 				      buildVarRegion, buildVarData)
 from functools import partial
 from collections import defaultdict
-from array import array
```

Comment:
is that because you found out `list` are actually faster than `array.array`?

Reply:
Yes. I will comment in https://github.com/fonttools/fonttools/issues/2267
But in general, in CPython, they super-optimized list and tuple. Anything else is lots slower :(.
Even the use in `GlyphCoordinates` I think I will remove soon.